.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # testing
 /coverage
+/cypress/integration
```

Comment:
Ignoring the cypress directory because it has some a11y issues and some other bits.